library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -627,7 +627,14 @@ libspdm_return_t libspdm_build_response(void *spdm_context, const uint32_t *sess
                        libspdm_get_scratch_buffer_large_message_offset(spdm_context);
         large_buffer_size = libspdm_get_scratch_buffer_large_message_capacity(spdm_context);
 
-        if (my_response_size < large_buffer_size) {
+        if (my_response_size > context->connection_info.capability.max_spdm_msg_size) {
+            status = libspdm_generate_extended_error_response(context,
+                                                              SPDM_ERROR_CODE_RESPONSE_TOO_LARGE,
+                                                              0,
+                                                              sizeof(my_response_size),
```

Comment:
I think maybe `sizeof(my_response_size)` at line 634 should be changed to`sizeof(uint32_t)`.
Because `ExtendedErrorData` **length** for `ResponseTooLarge` is explicitly 4 in `Table 52 — ExtendedErrorData format for ResponseTooLarge` in DSP0274_1.2.1.pdf

Reply:
Agree with Zhiqiang.

Reply:
Yes. I have added the `actual_size` to ensure that the code is right whenever using in big endian or little endian.